### PR TITLE
[icons]: team, trophy, soccer ball, sports stadium

### PR DESCRIPTION
**Pull request by jessperrin**
#### Checklist

- [ ] Includes tests
- [ ] Update documentation

#### Changes proposed in this pull request:

Soccer-based icons for an sports pilot.

image

### CODE REVIEW

**Comment by adidahiya** [comment]
<h3>Clean up PR a bit</h3>
Build artifact links for this commit: <strong><a href="https://output.circle-artifacts.com/output/job/3a93f30a-d0e9-4907-a797-a152fa912d42/artifacts/0/packages/docs-app/dist/index.html">documentation</a> | <a href="https://output.circle-artifacts.com/output/job/3a93f30a-d0e9-4907-a797-a152fa912d42/artifacts/0/packages/landing-app/dist/index.html">landing</a> | <a href="https://output.circle-artifacts.com/output/job/3a93f30a-d0e9-4907-a797-a152fa912d42/artifacts/0/packages/table-dev-app/dist/index.html">table</a> | <a href="https://output.circle-artifacts.com/output/job/3a93f30a-d0e9-4907-a797-a152fa912d42/artifacts/0/packages/demo-app/dist/index.html">demo</a></strong>

<em>This is an automated comment from the deploy-preview CircleCI job.</em>

File: resources/icons/16px/sports-stadium.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_1_17)">
```
Comment (adidahiya) [inline resources/icons/16px/sports-stadium.svg]: I think the `<g>` wrapper and `<defs>` are superfluous, they should probably be removed from this file

File: resources/icons/20px/sports-stadium.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (adidahiya) [inline resources/icons/20px/sports-stadium.svg:1]: this version of the sports-stadium icon markup looks better :+1:

File: resources/icons/20px/trophy.svg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_1_13)">
```
Comment (adidahiya) [inline resources/icons/20px/trophy.svg]: same here, we should remove the `<g>` wrapper and the `<defs>` block

**Review by adidahiya — CHANGES_REQUESTED** [review]
most of the icons look fine but 2 of them have extra markup that causes them to render incorrectly:

image

image

**Comment by jessperrin** [comment]
Oh hm, would you mind making those changes? Not sure why those are there - is it due to something in the icon SVG?

**Comment by adidahiya** [comment]
@jessperrin I'm not sure either, probably something to do with Figma export. Sure, I can make those fixes

**Comment by adidahiya** [comment]
<h3>remove extraneous SVG markup</h3>
Build artifact links for this commit: <strong><a href="https://output.circle-artifacts.com/output/job/7c5aa3e4-eaf3-44ac-818c-95162bc6f1b9/artifacts/0/packages/docs-app/dist/index.html">documentation</a> | <a href="https://output.circle-artifacts.com/output/job/7c5aa3e4-eaf3-44ac-818c-95162bc6f1b9/artifacts/0/packages/landing-app/dist/index.html">landing</a> | <a href="https://output.circle-artifacts.com/output/job/7c5aa3e4-eaf3-44ac-818c-95162bc6f1b9/artifacts/0/packages/table-dev-app/dist/index.html">table</a> | <a href="https://output.circle-artifacts.com/output/job/7c5aa3e4-eaf3-44ac-818c-95162bc6f1b9/artifacts/0/packages/demo-app/dist/index.html">demo</a></strong>

<em>This is an automated comment from the deploy-preview CircleCI job.</em>

**Review by adidahiya — APPROVED** [review]

**Pull request merged**